django_ractjs/reactjs_django/src/components/Extraction.js

```diff
@@ -0,0 +1,80 @@
+import { Button, Form, Modal, Row, Col, InputGroup } from "react-bootstrap";
+import { useState } from "react";
+import axios from "axios";
+
+export function Extraction({
+  showModal,
+  updateModal,
+  columnNames,
+  title,
+  url,
+}) {
+  const [isSaving, setIsSaving] = useState(false);
+
+  const [columnNamesArray, setColumnNamesArray] = useState([]);
+
+  const handleClose = () => updateModal(false);
+
+  let api = "http://127.0.0.1:8000/api";
+
+  const handleFilterColumnNames = (event, option) => {
+    const isChecked = event.target.checked;
+
+    if (isChecked) {
+      columnNamesArray.push(option);
+    } else {
+      //Code to be written
```

Comment:
@Utkrist1508 The code is to written here. Add the logic to add or remove from the array based on the whether the checkbox is active or inactive. Just add the Javascript logic

Reply:
Use the add and delete part and all the corner cases

Reply:
Done as aspected ...pls check & revert if found any issue